lib/parseConfig/getFunctionParameter.js

```diff
@@ -0,0 +1,24 @@
+/*
+* Given a function parameter, return a function that can be run at runtime
+* which evaluates the function string, and errors if required.
+*/
+var when = require('when');
+var _ = require('lodash');
+
+module.exports = function (parameter, key) {
+	return function () {
+		try {
+			eval('var evaluatedFn = '+parameter.value);	
```

Comment:
Because this doesn't work (try it in the browser):

```
eval("function () { return 'chris'; }")
```

FYI - all function values will be stored in this function string format.